include/aspect/mesh_deformation/interface.h

```diff
@@ -87,17 +87,30 @@ namespace aspect
          */
         virtual void update();
 
+        /**
+        * A function that returns the initial deformation of certain mesh
+        * vertices (e.g. the surface vertices). @p position is the original
```

Comment:
We currently use a MappingQ1Eulerian meaning the positions are indeed the vertices, but this is not necessarily the case in the future. Does it make sense to make this comment more general and just state that this will be evaluated on various points on the boundary?